### DISCUSSION TITLE
metabolite formation

### DISCUSSION THREAD

**Discussion by Nathalie06** [Open-Systems-Pharmacology/Forum#783] [Questions & Problems] [answered]
Hi all,

I am sorry, it is probably a stupid mistake but I can't see why the metabolite is not formed.
I added all the CYP in the population, I assigned the name of the metabolite for the pathways forming the metabolite, and I have no metabolite formation.
My compounds are OK, because they have already been used in another project. 
I use PK-Sim version 8.

Thanks in advance, 
Nathalie

I have attached my project
[D_MTB.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/5237827/D_MTB.zip)

**Comment by Nathalie06** [comment]
I think it is due to the population I have used, there is no expression indicated because the database is not available anymore. Individual available from templates are very limited (European (P-gp modified, CYP3A4 36 h), Japanese (Yasui studies) and Standard european male for DDI). 
Could it be due to the new available version of PK-sim ?

**Comment by msevestre** [comment]
@Nathalie06 
>because the database is not available anymore.

What database are you referring to?

**Comment by Nathalie06** [comment]
The database for expression (CYP, trasporters, protein binding partners)

**Comment by msevestre** [comment, marked as answer]
The database is available. When going from one version to another, the link should be copied over. But if not, you need to set it in your settings
This will not explain however why you have no metabolite :) unless you did not define any enzymatic activity.

see https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-options#template-database-path

**Comment by Nathalie06** [comment]
Thanks @msevestre !
I reload the database, I have "lost" it. 
It explains why I have no metabolite formation because the CYP expression was always 0% (even with a CYP3A4 pathways defined, there is no CYP3A4 expressed, so no metabolism and no metabolite formation).

Thanks again !

**Comment by msevestre** [comment]
@Nathalie06  Glad it worked out. It's not always so easy :)
